src/pages/settings/Subscription/SubscriptionDetails/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import Icon from '@components/Icon';
+import * as Illustrations from '@components/Icon/Illustrations';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import type {OptionsPickerItem} from '@components/OptionsPicker';
+import OptionsPicker from '@components/OptionsPicker';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+type SubscriptionVariant = ValueOf<typeof CONST.SUBSCRIPTION.TYPE>;
+
+const options: Array<OptionsPickerItem<SubscriptionVariant>> = [
+    {
+        key: CONST.SUBSCRIPTION.TYPE.ANNUAL,
+        title: 'subscription.details.annual',
+        icon: Illustrations.SubscriptionAnnual,
+    },
+    {
+        key: CONST.SUBSCRIPTION.TYPE.PAYPERUSE,
+        title: 'subscription.details.payPerUse',
+        icon: Illustrations.SubscriptionPPU,
+    },
+];
+
+function SubscriptionDetails() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const [privateSubscription] = useOnyx(ONYXKEYS.NVP_PRIVATE_SUBSCRIPTION);
+    const [account] = useOnyx(ONYXKEYS.ACCOUNT);
+    const [preferredTheme] = useOnyx(ONYXKEYS.PREFERRED_THEME);
```

Comment:
Why did you use `useOnyx` directly here instead of `useTheme`? Theme context would probably be more performant in general, plus it's just more consistent across the codebase.